cache/cache.cpp

```diff
@@ -555,25 +559,24 @@ static int dnet_cmd_cache_io_lookup_new(struct cache_manager *cache,
 		}
 	}
 
-	auto response = serialize(dnet_lookup_response{
-		0, // record_flags
-		it.user_flags, // user_flags
-		"", // path
+	std::unique_ptr<n2::lookup_response>
+	response(new n2::lookup_response(request->cmd,
```

Comment:
It's strange style. Please, fix everywhere:
```
	std::unique_ptr<n2::lookup_response> response(
		new n2::lookup_response(request->cmd,
```
or
```
	std::unique_ptr<n2::lookup_response>
		response(new n2::lookup_response(request->cmd,
```

Reply:
Fixed.